### ISSUE TITLE
Document how to write Columns

### ISSUE THREAD

**Issue by bsipocz** [astropy/astropy#3214]
This would have been useful for me a few times in the past, and I have a half working branch for it, too (reading is not working in the way I though, but I think it should be something trivial I don't see). 
The question whether it is desired to have, or the workaround to convert the given Column to a new Table is the preferred approach.

cc: @taldcroft

**Comment by bsipocz** [comment]
From time to time I run into this that it would be so convenient to be able to write a column without putting in into a Table. 
But @mhvk has a good point above, so I'm changing this issue from a feature request to a docs question. However not sure what the best place for the example above would be in the documentation.

**Comment by taldcroft** [comment]
Yeah, it's always tough to figure out where to put a little thing like that in the docs so it is discoverable.  We could make a Tips and Tricks section, but I suspect it will end up having one entry for a long time...  I almost think the best solution is for you to post the question on SO and I'll answer it.  Then a user googling will find it.

**Comment by mhvk** [comment]
There certainly is room for elaboration in the table I/O section: http://docs.astropy.org/en/latest/table/io.html

On the main docs page, the current "Files, I/O, and Communication" only discusses tables. It might make sense to have a "Getting started" section that discusses for all the various astropy classes how to save them (certainly should include `NDData`!).

**Comment by hamogu** [comment]
It's marked "package-novice" since everything is essentially spelled out above, but it's be more than a decade that this issue was opened, so I think we've been waiting long enough for a package-novice - at some point someone just has to do it.
